src/main/java/teammates/ui/webapi/output/CoursesData.java

```diff
@@ -0,0 +1,21 @@
+package teammates.ui.webapi.output;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+
+/**
+ * The API output format of a list of {@link CourseAttributes}.
+ */
+public class CoursesData extends ApiOutput {
```

Comment:
Same here.

Reply:
This appears to be used in GET courses, which is outside the scope of this PR?

Reply:
Notice that. You can change it in your phase 2 PR.